src/components/Finance/Finance.js

```diff
@@ -33,27 +33,20 @@ const useStyles = makeStyles((theme) => ({
 function Finance() {
   const classes = useStyles();
   const [dataLoading, setDataLoading] = useState(true);
-  const [currentUserData, setCurrentUserData] = useState();
   const [update, setUpdate] = useState(false);
-  const [financeData, isLoading] = usePageData("Finance", update);
+  const [pageData, isLoading] = usePageData("Finance", update);
   const [openFinancePopup, setOpenFinancePopup] = useState(false);
   const [multipleSelected, setMultipleSelected] = useState(false);
-  const [sortedData, setSortedData] = useState(null);
-  const [settlementData, setSettlementData] = useState(null);
+  const [sortedData, setSortedData] = useState([]);
```

Comment:
Falls ich das verwendet hatte, hatte glaube ich iwo mal auf null geprüft - funktioniert noch alles? :)

Reply:
sollte in Finance/55 passieren